src/Symfony/Component/Form/Extension/Core/Type/TelType.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Extension\Core\Type;
+
+use Symfony\Component\Form\AbstractType;
+
+class TelType extends AbstractType
```

Comment:
Last try.. :)

Given the SF form type refers the HTML spec, it doesnt add much value (semantically) when using forms without rendering.. right? I could simply go with `TextType` as well.

Meaning `UrlType`, `EmailType` also refer HTML spec. However `UrlType` adds value due `FixUrlProtocolListener` but that's is not related to the HTML spec at all.

TLDR; i end up with;

```php
$builder
  ->add('email', TextType::class)
//->add('email', EmailType::class) // better semantics
  ->add('url', UrlType::class) // handles protocols
  ->add('phone', TextType::class)
//->add('phone', TelType::class) // worse semantics
```

I really dont see how a form definition relates to some HTML spec.